base_business_document_import/models/business_document_import.py

```diff
@@ -71,12 +71,13 @@ def _match_partner(self, partner_dict, chatter_msg, partner_type="supplier"):
         company_id = self._context.get("force_company") or self.env.user.company_id.id
         domain = ["|", ("company_id", "=", False), ("company_id", "=", company_id)]
         if partner_type == "supplier":
-            domain += [("supplier_rank", ">", 0)]
+            order = "supplier_rank desc"
             partner_type_label = _("supplier")
         elif partner_type == "customer":
-            domain += [("customer_rank", ">", 0)]
+            order = "customer_rank desc"
             partner_type_label = _("customer")
         else:
+            order = ""
```

Comment:
Is `order = ""` working fine? Not crashing? :thinking:

Reply:
I just tested in a shell not crash